client/src/actions/roundApplication.ts

```diff
@@ -270,7 +275,9 @@ export const submitApplication =
         roundAddress,
         projectId: projectID,
       });
-      dispatch<any>(fetchProjectApplications(projectID, chainID));
+      dispatch<any>(
+        fetchProjectApplications(projectID, Number(projectChainId), process.env)
+      );
     } catch (e) {
       datadogRum.addError(e);
       console.error("error calling applyToRound:", e);
```

Comment:
also here we could remove the console.error no? Also on a side note, when we dispatch the applicationError(li 285) we don't actually send the actual error, just the status and generic message.

Reply:
Refactoring a bit